packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -181,24 +189,55 @@ export async function getPortalsTradeQuote(
         feePercentage: affiliateBpsPercentage,
         validate: false,
         swapperConfig,
+      }).catch(e => {
+        console.info('failed to get Portals quote with validation disabled', e)
+        return undefined
       })
 
-      if (dummyOrderResponse?.maybeGasLimit)
+      if (quoteEstimateResponse && !order) {
```

Comment:
I am never seeing this condition hit which would negate the need for all of the response payload checks below. Curious what flow ended up here. I am getting an order both with and without a wallet.

Reply:
Cleaned up logic in https://github.com/shapeshift/web/commit/43fe29b9a829a28688c10c66e1cd68c5cdabb6e9 with a clear branch in case of `!hasWallet`, should be much cleaner now!